packages/split-view/src/SplitView.ts

```diff
@@ -0,0 +1,416 @@
+    @property({ type: Number, attribute: 'secondary-min' })
+    public secondaryMin = 304;
+
+    /** The maximum size of the secondary pane */
+    @property({ type: Number, attribute: 'secondary-max' })
+    public secondaryMax = Infinity;
+
+    @property({ type: Number, attribute: false })
+    public dividerPosition = this.primaryDefault;
+
+    @property({ type: Number, attribute: false })
+    public minPos = 0;
+
+    @property({ type: Number, attribute: false })
+    public maxPos = Infinity;
+
+    @property({ type: Boolean, reflect: true, attribute: 'is-collapsed-start' })
+    public isCollapsedStart = this.dividerPosition === 0;
+
+    @property({ type: Boolean, reflect: true, attribute: 'is-collapsed-end' })
+    public isCollapsedEnd = false;
+
+    @property({ type: Boolean, reflect: true })
+    public dragging = false;
+
+    @property({ type: Boolean, reflect: true })
+    public hovered = false;
+
+    @property()
+    public label?: string;
+
+    private offset = 0;
+    private size = 0;
+    private isOver = false;
+    private lastPosition = 0;
+
+    public constructor() {
+        super();
+        this.onPointerDragged = this.onPointerDragged.bind(this);
+        this.onPointerUp = this.onPointerUp.bind(this);
+        this.resize = this.resize.bind(this);
+        this.updatePosition = this.updatePosition.bind(this);
+        this.getSplitterSize = this.getSplitterSize.bind(this);
+    }
+
+    public connectedCallback(): void {
+        super.connectedCallback();
+        window.addEventListener('resize', this.resize);
+    }
+
+    public disconnectedCallback(): void {
+        window.removeEventListener('resize', this.resize);
+        super.disconnectedCallback();
+    }
+
+    protected render(): TemplateResult {
+        const dimension = this.vertical
+            ? ORIENTATIONS.vertical
+            : ORIENTATIONS.horizontal;
+
+        return html`
+            <div
+                class="pane"
+                style=${`${dimension}: ${this.dividerPosition}px`}
+            >
+                <slot name="primary"></slot>
+            </div>
+            <div
+                id="splitter"
+                role="separator"
+                aria-label=${ifDefined(this.label || undefined)}
+                tabindex=${ifDefined(this.resizable ? '0' : undefined)}
+                @keydown=${this.onKeyDown}
+            >
+                ${this.resizable
+                    ? html`
+                          <div id="gripper"></div>
+                      `
+                    : html``}
+            </div>
+            <div class="pane secondaryPane">
+                <slot name="secondary"></slot>
+            </div>
+        `;
+    }
+
+    private onPointerMove(event: PointerEvent): void {
+        this.isOver = true;
+        if (this.dragging) {
+            return;
+        }
+        this.updateCursor(event);
+    }
+
+    private onPointerDown(event: PointerEvent): void {
```

Comment:
Nit, this event is `pointerdown`. Please only camel case the `P`, both here and on others.

Reply:
fixed